index.html

```diff
@@ -128,36 +167,38 @@ <h3>LET'S CONNECT</h3>
             <li><a href="#"><img src="images/tech.png" alt="Image"></a></li>
           </ul>
         </div>
-        <form action="#" class="Click" target="_blank" method="get">
-          <button class="button-2" type="button">Get my resume</button>
-        </form>
+      <form action="#" id="GE" class="Click" target="_blank" method="get">
+        <button class="button-2" type="button">Get my resume</button>
+      </form>
+    </div>
       <ul class="Skills">
         <li class="Skill-list">Languages <a href="#"><img class="Pointer-1" src="images/Union - Copy.png" alt="icon"></a>
-          <li id="JAVASCRIPT"><img src="images/Ellipse 1.png" alt="icon">Javascript</li>
-          <li id="HTML"><img src="images/html.png" alt="icon">HTML</li>
-          <li id="CSS"><img src="images/css (2).png" alt="icon"></IMg> CSS</li>
+          <ul class="Languages">
+            <li id="JAVASCRIPT"><img src="images/Ellipse 1.png" alt="icon">Javascript</li>
+            <li id="HTML"><img src="images/html.png" alt="icon">HTML</li>
+            <li id="CSS"><img src="images/css (2).png" alt="icon"></IMg> CSS</li>
+          </ul>
```

Comment:
- [ optional] Kindly note that according to HTML/CSS best practices, it would be great to use `class selector` for multiple elements not `id`. 😄